protocol/synthetix/storage.dump.sol

```diff
@@ -665,3 +676,8 @@ library CcipClient {
         bool strict;
     }
 }
+
+// @custom:artifact hardhat/console.sol:console
+library console {
```

Comment:
Can we do a yarn clean and yarn build to get this removed ( and fix wrong version number)